doc/install.rst

```diff
@@ -0,0 +1,57 @@
+Installation
+============
+
+This section provide guidelines for installing DataTransferKit and its TPLs.
+
+Install third-party libraries
+-----------------------------
+
+The following third party libraries (TPLs) are used by mfmg:
+
++------------------------+---------+
+| Packages               | Version |
++========================+=========+
+| ARPACK                 | N/A     |
++------------------------+---------+
+| Boost                  | 1.65.1  |
++------------------------+---------+
+| BLAS/LAPACK            | N/A     |
++------------------------+---------+
+| deal.II                | 8.5     |
++------------------------+---------+
+| MPI                    | N/A     |
++------------------------+---------+
+| Trilinos               | 12.X    |
++------------------------+---------+
+
+The dependencies of mfmg may be built using `Spack
+<https://github.com/llnl/spack>`_ package manager. You need to install the
+following package:
+
+.. code::
+
+    $ spack install dealii
```

Comment:
I remember you mentioning `dealii@develop`? That's what I've been installing. Or is it only to get some GPU in?

Reply:
It's only for GPU